src/main/java/com/dougnoel/sentinel/elements/WindowsElement.java

```diff
@@ -182,38 +182,22 @@ public Element click() {
 	 * @param yOffset int the pixels below the top edge of the element
 	 * @return Color the color at the specified pixel offset in the element.
 	 */
-	public Color getColor(int xOffset, int yOffset) {
-		Robot robot;
-		try {
-			robot = new Robot();
-		} catch (AWTException awte) {
-			throw new com.dougnoel.sentinel.exceptions.IOException(awte);
-		}
-
-		// Grab the pixel coordinates of the rectangle of the element, relative to the top-left corner of the screen. Positive X = right. Positive Y = down. 
-		// These coordinates are given in the "true" pixel count of the screen. That is, on a 3840 x 2160 pixel (4K) screen, the coordinates given here are in that domain.
-		var elementRectangle = element().getAttribute("BoundingRectangle").split(" ");
-		var topLeftX = Integer.parseInt(elementRectangle[0].split(":")[1]);
-		var topLeftY = Integer.parseInt(elementRectangle[1].split(":")[1]);
-		var width = Integer.parseInt(elementRectangle[2].split(":")[1]);
-		var height = Integer.parseInt(elementRectangle[3].split(":")[1]);
-
-		var elementCoords = new java.awt.Rectangle(topLeftX, topLeftY, width, height);
+	public Color getColorAtOffset(int xOffset, int yOffset) {
 		// Windows (the operating system) contains a scaling setting which affects the "screen resolution" that Robot reads. 
 		// Because Robot respects that setting but the above "BoundingRectangle" coordinates do not, the user must have 100% in that setting so the resolutions are over the same domain.
-		BufferedImage screenshot = robot.createScreenCapture(elementCoords);
-		File outputfile = new File("logs/bufferedimage.png");
+		BufferedImage screenshot = screenshotElement();
+
 		try {
-			ImageIO.write(screenshot, "png", outputfile);
+			FileManager.saveImage(null, "tempGetColorImage.png", screenshot);
 		} catch (IOException ioe) {
 			throw new com.dougnoel.sentinel.exceptions.IOException(ioe);
 		}
 
 		int argb = screenshot.getRGB(xOffset, yOffset);
-		double a = (argb >> 24) & 0xFF;
+		double a = ((argb >> 24) & 0xFF)/255.0; //The selenium color object will take any double for alpha, but requires 0.0-1.0 for correct operation. This converts the alpha from 0-255 to the correct range.
```

Comment:
Awesome catch here

Reply:
Debugging that was so...that was still funny when I realized it.